Musigwa/SendIT-API
# [WIP] [Feature #162110507] Protect all the routes endpoints

## What does this PR do (?)

It protects all of the routes with JWT unless the Sign-up & Sign-in.

## Description of Task to be completed (?)

The following Route(s) should be protected:

- GET /parcels
- GET /parcels/<parcel_id>
- GET /users/<user_id>/parcels
- PUT /parcels/<parcel_id>/cancel
- POST /parcels

The following Route(s) should be accessible without authentication:

- GET

## How should this be manually tested (?)

After cloning this repo and switch to this branch,

- Using any RESTful Client tool (Usually the postman), test every endpoint, make sure to set your Headers as the following
- The private Route(s) should return {message:access denied'}
- The public Route(s) be accessible and working as intended

Key: Content-Type value: application/json

## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. 162110507
